content/en/cloudcraft/getting-started/datadog-integration.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Datadog Integration
+---
+
+## Overview
+
+The integration between Datadog and Cloudcraft provides users with a streamlined workflow for monitoring and visualizing their cloud infrastructure.
+
+By leveraging Datadog’s powerful monitoring platform, users can log in to Cloudcraft with their Datadog account, move seamlessly from any resource in Cloudcraft to the most relevant views in Datadog, and automatically pull cloud accounts that have already been configured in Datadog for use in Cloudcraft.
+
+## Datadog Single Sign-On (SSO)
+
+Cloudcraft allows users to sign up and log in using their Datadog account. This integration offers a unified experience, linking your Datadog monitoring data with your Cloudcraft architecture diagrams.
+
+### Sign up with Datadog SSO
+
+To get started, choose the **Sign up with Datadog** option during the Cloudcraft sign-up process. After signing up, you can log in to Cloudcraft using your Datadog credentials. This simplifies the login process and enables integration between the two platforms.
+
+By using Datadog SSO, you automatically gain access to:
+
+- **Cross-platform functionality**: Move effortlessly between Cloudcraft and Datadog to analyze your cloud infrastructure and its performance.
+- **Automated cloud account synchronization**: Cloud accounts configured in Datadog sync automatically with Cloudcraft, giving you a complete view of your infrastructure in both platforms.
+
+### Enable Datadog SSO for existing accounts
+
+If you originally signed up with a different login method, for example, Google SSO or a standard username and password, you won’t have access to the full set of Datadog integration features. To switch to Datadog SSO, [contact the Cloudcraft support team][1] and they will assist you in converting your account.
+
+## Pull cloud accounts
+
+<div class="alert alert-info">This feature supports only Amazon Web Services (AWS) accounts. Syncing with Azure or other cloud providers is not available at this time.</div>
+
+The integration between Cloudcraft and Datadog streamlines cloud account management, allowing accounts already configured in Datadog to be automatically pulled into Cloudcraft. No additional setup is required in Cloudcraft as long as resource collection is enabled in Datadog.
+
+### How it works
+
+1. **Pulling cloud accounts**: Once your AWS account is integrated with Datadog, Cloudcraft automatically pulls the account, so you don’t need to manually add it to Cloudcraft.
```

Comment:
I am not sure if we need a separate section for how it works or if it can be just 2 small paragraphs under "Cloud Account Registration"

Consider changing "How it works" section into a second para talking about resource collection - something like:

To visualize and diagram resources in Cloudcraft, ensure resource collection (provide link) is enabled in Datadog.
When resource collection is enabled, Datadog collects information about your AWS resources by making read-only API calls to your AWS account. Cloudcraft relies on this information for visualizing your infrastructure.